Dockerfile

```diff
@@ -1,6 +1,5 @@
 FROM mozillabteam/bmo-slim:20170927.1
 
-ENV BUNDLE=https://s3.amazonaws.com/moz-devservices-bmocartons/bmo/vendor.tar.gz
```

Comment:
This change isn't documented as necessary anywhere in the PR or in the bug, but it's mixed in with a documented change below to add checksetup. Is this an intentional change?

Reply:
I'll add it back, it isn't used by anything but it was left over from another session.